doc/man3/OSSL_PARAM.pod

```diff
@@ -160,6 +160,7 @@ systems.
 =item B<OSSL_PARAM_REAL>
 
 The parameter data is a floating point value in native form.
+The OpenSSL library only implements the C<double> type.
```

Comment:
What about this suggestion:

`The OpenSSL providers and helper functions support only C<double> sized values.`

Reply:
That would be better, yeah...  I gotta wonder, though, why this particular data type is singled out.

Reply:
I took @t8m's change, and now I'm done.  Merge or close this, and close or leave open Kurt's issue... whatever.  Time to move on.